client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-top-header-buttons/index.tsx

```diff
@@ -0,0 +1,110 @@
+import { isEnabled } from '@automattic/calypso-config';
+import { Button, WordPressLogo } from '@automattic/components';
+import { useMobileBreakpoint } from '@automattic/viewport-react';
+import classNames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import { useRef, useState } from 'react';
+import JetpackLogo from 'calypso/components/jetpack-logo';
+import PopoverMenuItem from 'calypso/components/popover-menu/item';
+import SplitButton from 'calypso/components/split-button';
+import { useDispatch } from 'calypso/state';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import WPCOMHostingPopover from './wpcom-hosting-popover';
+
+export default function SiteTopHeaderButtons() {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+	const isMobile = useMobileBreakpoint();
+
+	const isWPCOMAtomicSiteCreationEnabled = isEnabled(
+		'jetpack/pro-dashboard-wpcom-atomic-hosting'
+	);
+
+	const buttonRef = useRef< any | null >( null );
+	const [ toggleIsOpen, setToggleIsOpen ] = useState( false );
+
+	return (
+		<div
+			className={ classNames( 'sites-overview__add-site-issue-license-buttons', {
+				'is-with-split-button': isWPCOMAtomicSiteCreationEnabled,
+			} ) }
+		>
+			<Button
+				className="sites-overview__issue-license-button"
+				href="/partner-portal/issue-license"
+				onClick={ () =>
+					dispatch(
+						recordTracksEvent( 'calypso_jetpack_agency_dashboard_issue_license_button_click' )
+					)
+				}
+			>
+				{ translate( 'Issue License', { context: 'button label' } ) }
+			</Button>
+
+			{ isWPCOMAtomicSiteCreationEnabled ? (
+				<span>
+					<SplitButton
+						primary
+						whiteSeparator
+						label={ isMobile ? undefined : translate( 'Add new site' ) }
+						onClick={ () =>
+							dispatch(
+								recordTracksEvent(
+									'calypso_jetpack_agency_dashboard_create_wpcom_atomic_site_button_click'
```

Comment:
Do we need to track this event here? It looks like it has the same name as the event for the `Create a new WordPress.com site` button?

Reply:
Good catch. I copied and pasted this part of the code from a different component but didn't catch it. 

I have made the changes here: https://github.com/Automattic/wp-calypso/pull/81311/commits/ef3b52342fbf1524c9adc07e0473283438aa948b